server/src/ocv.rs

```diff
@@ -43,6 +43,116 @@ impl Ocv {
     Ok(ProposalResponse { proposal, votes })
   }
 
+  pub async fn proposal_consideration(&self, id: usize, start_time: i64, end_time: i64, ledger_hash: Option<String>) -> Result<GetMinaProposalConsiderationResponse> {
+    let proposal_key = "MEF".to_string() + &id.to_string();
+    let votes = if let Some(cached_votes) = self.caches.votes.get(&proposal_key).await {
+      cached_votes.to_vec()
+    } else {
+      let transactions = self.archive.fetch_transactions(start_time, end_time)?;
+
+      let chain_tip = self.archive.fetch_chain_tip()?;
+      let votes = Wrapper(transactions.into_iter().map(std::convert::Into::into).collect())
+      .process_mep(&id.to_string(), chain_tip)
+      .sort_by_timestamp()
+      .to_vec()
+      .0;
+
+      self.caches.votes.insert(proposal_key.clone(), Arc::new(votes.clone())).await;
+      tracing::info!("votes {}", votes.len());
+      votes
+    };
+  
+    // weighted votes
+    let mut positive_stake_weight = Decimal::from(0);
+    let mut negative_stake_weight = Decimal::from(0);
+
+    // check community votes
+    let mut total_positive_community_votes = 0;
+    let mut total_negative_community_votes = 0;
+    for vote in &votes {
+      if vote.memo.to_lowercase() == format!("yes id {}", id.to_string()) {
+        total_positive_community_votes += 1;
+      }
+      if vote.memo.to_lowercase() == format!("no id {}", id.to_string()) {
```

Comment:
same as above, `usize` implements Display
```suggestion
      if vote.memo.to_lowercase() == format!("no id {}", id) {
```

Reply:
fixed